examples/dreambooth/train_dreambooth.py

```diff
@@ -454,6 +454,7 @@ def __init__(
         tokenizer,
         class_data_root=None,
         class_prompt=None,
+        class_num=None,
```

Comment:
This should not be `None` actually, otherwise the `min` call will fail. When it's `None` we should set it to the value of `len(self.class_images_path)`

Reply:
Yeah, that's my mistake. Change the code like this:
```py
if class_num is not None:
    self.num_class_images = min(len(self.class_images_path), class_num)
else:
    self.num_class_images = len(self.class_images_path)
```